### PR TITLE
Improves developers getting started guide

### PR DESCRIPTION
**Pull request by fowlslegs**
The Mac corrections are based on what I learned helping @ninavizz get setup on
her Mac. The Debian corrections are based on what I know from my own system.

### CODE REVIEW

File: docs/development/getting_started.rst [hunk 1]
```diff
@@ -78,8 +78,14 @@ Ansible. Using `virtualenvwrapper
...
     sudo apt-get install virtualenvwrapper
-    mkvirtualenv -p python2.7 securedrop
-    pip install ansible==1.8.4
+    source /usr/share/virtualenvwrapper/virtualenvwrapper.sh
+    mkvirtualenv -p /usr/bin/python2 securedrop
+    pip install -U ansible==1.8.4
+
+.. note:: You'll want to add the command to source ``virtualenvwrapper.sh``
+          to your ``~/.bashrc`` (or whatever your default shell configuration
+          file is) so that the commandline utilites ``virtualenvwrapper``
```
Comment (redshiftzero) [inline docs/development/getting_started.rst]: 'commandline' -> 'command line'

File: docs/development/getting_started.rst [hunk 2]
```diff
@@ -90,17 +96,25 @@ Install the dependencies for the development environment:
...
+    source /usr/local/bin/virtualenvwrapper.sh
+    mkvirtualenv -p python2 securedrop
+    pip install -U ansible==1.8.4
 
-.. note:: If you install ``virtualenvwrapper`` and get a ``mkvirtualenv: 
-          command not found`` error, make sure 
-          ``source /usr/local/bin/virtualenvwrapper.sh`` is in your ``~/.bashrc``.
+.. note:: You'll want to add the command to source ``virtualenvwrapper.sh``
+          to your ``~/.bashrc`` (or whatever your default shell configuration
+          file is) so that the commandline utilites ``virtualenvwrapper``
```
Comment (redshiftzero) [inline docs/development/getting_started.rst]: 'commandline' -> 'command line'

Comment (redshiftzero) [inline docs/development/getting_started.rst]: 'utilites' -> 'utilities'

File: docs/development/getting_started.rst (hunk 1 above)
Comment (redshiftzero) [inline docs/development/getting_started.rst]: 'utilites' -> 'utilities'

Comment (fowlslegs) [inline docs/development/getting_started.rst]: Wikipedia uses "command-line," which I like.

**Comment by fowlslegs** [comment]
Changes made.

File: docs/development/getting_started.rst [hunk 3]
```diff
@@ -90,17 +96,25 @@ Install the dependencies for the development environment:
 #. VirtualBox_
 #. Ansible_.
 
-There are several ways to install Ansible on a Mac. We recommend setting up a virtual environment to install Ansible since we are currently using Ansible 1.8.4:
+There are several ways to install Ansible on a Mac. We recommend installing it
+to a virtual environment using ``virtualenvwrapper`` and `pip`, so as not to
```
Comment (redshiftzero) [inline docs/development/getting_started.rst]: sorry just caught this when doing a test build of the docs: double backtick for fixed width font needed on ``pip``:

screen shot 2017-02-01 at 5 48 51 pm

Comment (fowlslegs) [inline docs/development/getting_started.rst]: Fixed!

**Pull request merged**